Automattic/wp-calypso
# Remove unused routes and code, styles behind those routes from previous iterations of the Jetpack Checkout

#### Changes proposed in this Pull Request

This PR removes some unused routes and the associated code, styles of those routes from previous iterations of the Jetpack Checkout

These changes remove the unused Jetpack site-less purchase, which was later replaced with the current flow based on user licenses.

#### Testing instructions

- Checkout to `fix/remove-unused-routes` branch and take a pull
- Start Calypso with yarn start.
- Visit https://cloud.jetpack.com/pricing.
- Select any Jetpack product other than Jetpack Search.
- Once you are on the Checkout page, replace https://wordpress.com with http://calypso.localhost:3000 leaving the rest of the URL untouched.
- Check out the product.
- Verify that you land on the activation flow where you can select one of your sites from a dropdown or follow the manual activation flow that ends up with your license key.

also 

- Verify that `jetpack-checkout-siteless-thank-you-completed.tsx` & `jetpack-checkout-siteless-thank-you.tsx` are deleted and the references are removed
- Verify that styles that belong to `jetpack-checkout-siteless-thank-you-completed` and `jetpack-checkout-siteless-thank-you` are removed
- Verify that pages for `/checkout/jetpack/schedule-happiness-appointment`, `/checkout/jetpack/thank-you-completed/no-site/:product` & `/checkout/jetpack/thank-you/no-site/:product` are removed.

Related to 1164141197617539-as-1201624154131998
